cmake_modules/FindHWLOC.cmake

```diff
@@ -35,7 +35,7 @@ pkg_check_modules(PC_HWLOC QUIET hwloc)
 set(HWLOC_DEFINITIONS ${PC_HWLOC_CFLAGS_OTHER} )
 
 find_path(HWLOC_INCLUDE_DIR hwloc.h
-          HINTS ${PC_HWLOC_INCLUDEDIR} ${PC_HWLOC_INCLUDE_DIRS}
+          HINTS ${PC_HWLOC_INCLUDE_DIRS}
```

Comment:
According to cmake pkgconfig, *_INCLUDEDIR is the package directory the place most likely to contain the main header (aka hwloc.h). Why removing it from the hints ?